runelite-client/src/main/java/net/runelite/client/plugins/friendchat/FriendChatPlugin.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018, Connor <contact@connor-parks.email>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package net.runelite.client.plugins.friendchat;
+
+import com.google.common.eventbus.Subscribe;
+import java.util.Arrays;
+import javax.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.VarPlayer;
+import net.runelite.api.events.GameTick;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+
+@PluginDescriptor(
+	name = "Friend Chat",
+	description = "Add extra information to the friends chat"
+)
+@Slf4j
+public class FriendChatPlugin extends Plugin
+{
+	@Inject
+	private Client client;
+
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		Widget friendListTitleWidget = client.getWidget(WidgetInfo.FRIEND_CHAT_TITLE);
+		if (friendListTitleWidget == null)
+		{
+			return;
+		}
+
+		final int world = client.getWorld();
+		final long friends = Arrays.stream(client.getFriends()).filter(friend -> friend != null).count();
+		final int limit = client.getVar(VarPlayer.MEMBERSHIP_DAYS) > 0 ? 400 : 200;
+
+		final String title = new StringBuilder("Friends - W")
+			.append(world)
+			.append(" (")
+			.append(friends)
+			.append("/")
+			.append(limit)
+			.append(")")
+			.toString();
+
+		friendListTitleWidget.setText(title);
```

Comment:
You should revert the widget to original text on shutdown

Reply:
Which would make more sense here, do you think?
1) We set it to what it would be manually (ie build the "Friends List - World X" string;
2) We store the previous value on GameStateChange and re-set that in onShutdown.

Being that it's a simple fixed format, 1 makes most sense to me but I'm wondering what you'd prefer.

Reply:
I think we do 1. with combat info plugin, so probably that, as its really simple scenario.

Reply:
If it's set manually, wouldn't that persist after a world hop?

Reply:
This has now been addressed.